TombEngine/Objects/TR3/Entity/SophiaLeigh.cpp

```diff
@@ -582,9 +576,9 @@ namespace TEN::Entities::Creatures::TR3
 				{
 					item.Animation.TargetState = SOPHIALEIGH_STATE_STAND;
 
-					StopSoundEffect(SFX_TR3_SOFIALEIGH_SUMMON);
-					SoundEffect(SFX_TR3_SOFIALEIGH_SUMMON_FAIL, &item.Pose);
-					SoundEffect(SFX_TR3_SOFIALEIGH_TAKE_HIT, &item.Pose);
+					StopSoundEffect(SFX_TR3_SOFIALEE_SUMMON);
+					SoundEffect(SFX_TR3_SOFIALEE_SUMMON_FAIL, &item.Pose);
+					SoundEffect(SFX_TR3_SOFIALEE_TAKE_HIT, &item.Pose);
```

Comment:
I'm getting something lost here.

In the new Sophia's branch, is renaming the sounds from "SFX_TR3_SOFIALEIGH_SUMMON" to "SFX_TR3_SOFIALEE_SUMMON" but when I check in my current master branch, I'm finding "SFX_TR3_SOFIALEIGH_SUMMON"

Why is returning to the Lee?

Reply:
hmm merge error?

Reply:
Fixed thanks :)